1 Presentation layer/CandidateEvaluator.Client/Pages/Category/ListCategories.cshtml

```diff
@@ -22,6 +23,7 @@ else
                 <tr>
                     <td>@model.Id</td>
                     <td><a href="/category/@model.Id">@model.Name</a></td>
+                    <td><a href="/category/@model.Id/questions"> Questions </a></td>
```

Comment:
Maybe here you can place a number of questions?

Reply:
If you are okay with that I will open issue to create additional query for number to get number of qurstions. At this time we do not store this value anywhere